packages/serverpod/lib/src/database/expressions.dart

```diff
@@ -270,6 +270,34 @@ class ColumnByteData extends Column {
   ColumnByteData(String name) : super(name, ByteData);
 }
 
+/// A [Column] holding [Duration].
+class ColumnDuration extends Column {
+  /// Creates a new [Column], this is typically done in generated code only.
+  ColumnDuration(String name) : super(name, Duration);
+
+  /// Creates an [Expression] checking if the value in the column equals the
+  /// specified value.
+  Expression equals(Duration? value) {
+    if (value == null) {
+      return Expression('"$columnName" IS NULL');
+    } else {
+      return Expression(
+          '"$columnName" = ${DatabasePoolManager.encoder.convert(value)}');
```

Comment:
In these cases, the codes becomes a little bit more readable if you add a trailing comma at the end of the string.

Reply:
Ill add a trailing comma at the end of the string in each Expression